The failure seems unrelated to the PR:

```
2022-05-24T21:54:20.3789520Z [ERROR] io.quarkus.cli.CliProjectJBangTest.testCreateAppDefaults  Time elapsed: 1.024 s  <<< FAILURE!
2022-05-24T21:54:20.3790137Z org.opentest4j.AssertionFailedError: 
2022-05-24T21:54:20.3790798Z Expected OK return code. Result:
2022-05-24T21:54:20.3791172Z result: {
2022-05-24T21:54:20.3791685Z   exitCode: {1},
2022-05-24T21:54:20.3792071Z   system_err: {Downloading JBang...
2022-05-24T21:54:20.3792647Z Error downloading JBang
2022-05-24T21:54:20.3792993Z },
2022-05-24T21:54:20.3793489Z   system_out: {}
2022-05-24T21:54:20.3793875Z } ==> expected: <0> but was: <1>
```

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -79,8 +79,7 @@
         <plexus-utils.version>3.3.0</plexus-utils.version>
         <plexus-component-annotations.version>2.1.0</plexus-component-annotations.version>
         <graal-sdk.version>22.1.0</graal-sdk.version>
-        <!-- Use previous graal version till https://github.com/oracle/graal/issues/4518 gets resolved -->
-        <graal-svm.version>22.0.0.2</graal-svm.version>
+        <graal-svm.version>${graal-sdk.version}</graal-svm.version>
         <gizmo.version>1.0.11.Final</gizmo.version>
         <jackson-bom.version>2.13.3</jackson-bom.version>
         <commons-logging-jboss-logging.version>1.0.0.Final</commons-logging-jboss-logging.version>

